dynamic/src/util.rs

```diff
@@ -154,6 +154,30 @@ pub fn get_active_battle_object_id_from_entry_id(entry_id: u32) -> Option<u32> {
     }
 }
 
+/// includes nana
+pub unsafe fn get_all_player_battle_object_ids() -> Vec<u32> {
```

Comment:
does this also work for aegis, since they are technically different battle objects too?

Reply:
If I'm correct, it gets the active battle object, which is whichever of the two characters is currently out. Works for use cases of this function because you wouldn't need both. This should be tested.